src/devices/Kt0803/Country.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Kt0803
+{
+    /// <summary>
+    /// Country
+    /// </summary>
+    public enum Country
```

Comment:
Was this meant to be continent/region?

Reply:
also there should be option like Other for people from other Asian countries and Africa

Reply:
In fact, this setting is the `Pre-Emphasis Time Constant`. This parameter varies from country to country. Enum values come from the datasheet. I'll rename it to Region and add Other option.